js/index.js

```diff
@@ -40,3 +40,62 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let logo2 = document.getElementById("cta-img");
+logo2.setAttribute('src', siteContent["cta"]["img-src"])
+
+let logo3 = document.getElementById("middle-img");
+logo3.setAttribute('src', siteContent["main-content"]["middle-img-src"]);
+
+let navlinks = document.getElementsByTagName("a");
+
+navlinks[0].textContent = (siteContent["nav"]["nav-item-1"]);
+navlinks[1].textContent = (siteContent["nav"]["nav-item-2"]);
+navlinks[2].textContent = (siteContent["nav"]["nav-item-3"]);
+navlinks[3].textContent = (siteContent["nav"]["nav-item-4"]);
+navlinks[4].textContent = (siteContent["nav"]["nav-item-5"]);
+navlinks[5].textContent = (siteContent["nav"]["nav-item-6"]);
+
+let ctah1 = document.querySelectorAll("h1");
+ctah1[0].textContent = (siteContent["cta"]["h1"]);
+
+let btn = document.querySelector("button");
+btn.textContent = (siteContent["cta"]["button"]);
+
+let txt = document.querySelectorAll("h4");
+let para = document.querySelectorAll("p")
+
+txt[0].textContent = (siteContent["main-content"]["features-h4"]);
+para[0].textContent = (siteContent["main-content"]["features-content"]);
+txt[1].textContent = (siteContent["main-content"]["about-h4"]);
+para[1].textContent = (siteContent["main-content"]["about-content"]);
+txt[2].textContent = (siteContent["main-content"]["services-h4"]);
+para[2].textContent = (siteContent["main-content"]["services-content"]);
+txt[3].textContent = (siteContent["main-content"]["product-h4"]);
+para[3].textContent = (siteContent["main-content"]["product-content"]);
+txt[4].textContent = (siteContent["main-content"]["vision-h4"]);
+para[4].textContent = (siteContent["main-content"]["vision-content"]);
+
+txt[5].textContent = (siteContent["contact"]["contact-h4"]);
+para[5].textContent = (siteContent["contact"]["address"]);
+para[6].textContent = (siteContent["contact"]["phone"]);
+para[7].textContent = (siteContent["contact"]["email"]);
+
+para[8].textContent = (siteContent["footer"]["copyright"]);
+
+let append = document.createElement('a');
+let linkText = document.createTextNode("Take Me Away");
```

Comment:
what's the difference between createTextNode and textContent?

Reply:
with createTextNode you can apply the variable basically anywhere you would like just by appending or prepending, vs textContent is editing a specific section where text would be present (an already existing element).

Reply:
also textContent overwrites all previous text nodes including the current textContent